Add a few missing env variables to "Environment Variables" page. (#724)
pulumi/pulumi-hugo: themes/default/content/docs/reference/cli/environment-variables.md

            PULUMI_AUTOMATION_API_SKIP_VERSION_CHECK
        </span>
    </dt>
    <dd>
        <p>
            Skips the minimum CLI version check used by Automation API to ensure compatibility. We do not recommend using this variable as it may result in unexpected behavior or confusing error messages from Automation API.
        </p>
        <pre><code class="text-xs">PULUMI_AUTOMATION_API_SKIP_VERSION_CHECK=true</code></pre>
    </dd>
    <dt>
        <span class="font-mono">
            PULUMI_ACCESS_TOKEN
        </span>
    </dt>
    <dd>
        <p>
            Set this environment variable to authenticate into the Pulumi Service backend and bypass the access
            token prompt when running {{% md %}}`pulumi login`{{% /md %}}.
        </p>
        <pre><code class="text-xs">PULUMI_ACCESS_TOKEN="your-access-token"</code></pre>
    </dd>
    <dt>
        <span class="font-mono">
            PULUMI_BACKEND_URL
        </span>
    </dt>
    <dd>
        <p>
            Set this environment variable to use a specified backend instead of the default backend.  See <a href="/docs/intro/concepts/state">State and Backends</a> for details on valid backend URLs.
        </p>

>         <pre><code class="text-xs">PULUMI_BACKEND_URL=s3://your-pulumi-state-bucket</code></pre>

Reviewer: ```suggestion
        <pre><code class="text-xs">PULUMI_BACKEND_URL="s3://your-pulumi-state-bucket"</code></pre>
```

    </dd>
    <dt>
        <span class="font-mono">
            PULUMI_CONFIG
        </span>
    </dt>
    <dd>
        <p>

>             Sets <a href="/docs/intro/concepts/config">configuration</a> for <a href="/docs/guides/testing/unit">unit testing</a>. Must be in JSON format.

Reviewer: ```suggestion
            Sets <a href="{{ relref . "/docs/intro/concepts/config"}}">configuration</a> for <a href="{{ relref . "/docs/guides/testing/unit" }}">unit testing</a>. Must be in JSON format.
```

        </p>
        <p>

>             <b>This environment variable is ignored during normal Pulumi operations - e.g. {{% md %}}`up`, `pre`, etc{{% /md %}}.</b>

Reviewer: ```suggestion
            <strong>This environment variable is ignored during normal Pulumi operations -- e.g., <code>up</code>, <code>preview</code>, etc.</strong>
```

        <p>
            Set this as an environment variable to protect and unlock your configuration values and secrets. Your passphrase
            is used to generate a unique key for your stack, and configuration and encrypted state values are then encrypted

>             using {{% md %}}`AES-256-GCM`{{% /md %}}.

Reviewer: ```suggestion
            using <code>AES-256-GCM</code>.
```

    </dd>
    <dt>
        <span class="font-mono">
            PULUMI_CONFIG_PASSPHRASE_FILE
        </span>
    </dt>
    <dd>
        <p>

>             An alternative method to providing {{% md %}}`PULUMI_CONFIG_PASSPHRASE`{{% /md %}}. Set this to a path to a file that contains the passphrase value.

Reviewer: ```suggestion
            An alternative method to providing <code>PULUMI_CONFIG_PASSPHRASE</code>. Set this to the path of a file that contains the passphrase value.
```

    </dd>
    <dt>
        <span class="font-mono">
            PULUMI_CONSOLE_DOMAIN
        </span>
    </dt>
    <dd>
        <p>
            Overrides the domain used when generating links to the Pulumi Console.
        </p>
        <pre><code class="text-xs">PULUMI_CONSOLE_DOMAIN="yourhost.domain.com"</code></pre>
    </dd>
    <dt>
        <span class="font-mono">
            PULUMI_DEBUG_PROMISE_LEAKS
        </span>
    </dt>
    <dd>
        <p>
            As of <a href="https://github.com/pulumi/pulumi/blob/master/CHANGELOG.md#0166-2018-11-28"><code>v0.12.2</code></a>,
            the promise leak experience has been improved and shows a simple error message. Set this environment variable to
            get more verbose error messages when debugging promise leaks.
        </p>
        <pre><code class="text-xs">PULUMI_DEBUG_PROMISE_LEAKS=true</code></pre>
    </dd>
    <dt>
        <span class="font-mono">
            PULUMI_ENABLE_LEGACY_APPLY
        </span>
    </dt>
    <dd>
        <p>
            As of <a href="https://github.com/pulumi/pulumi/blob/master/CHANGELOG.md#100-beta1-2019-08-13"><code>1.0.0-beta1</code></a>,

>             input properties are no longer propagated to missing output properties during a {{% md %}}`pulumi preview`{{% /md %}}. If this causes issues

Reviewer: ```suggestion
            input properties are no longer propagated to missing output properties during a <code>pulumi preview</code>. If this causes issues
```

>             in your Pulumi program, set this to {{% md %}}`true`{{% /md %}} to enable the old behavior.

Reviewer: ```suggestion
            in your Pulumi program, set this to <code>true</code> to enable the old behavior.
```

        </p>
        <pre><code class="text-xs">PULUMI_ENABLE_LEGACY_APPLY=true</code></pre>
    </dd>
    <dt>
        <span class="font-mono">
            PULUMI_ENABLE_LEGACY_DIFF
        </span>
    </dt>
    <dd>
        <p>
            As of <a href="https://github.com/pulumi/pulumi/blob/master/CHANGELOG.md#01723-2019-07-16"><code>v0.17.23</code></a>,
            the detection of differences between the actual and desired state of a resource is left entirely up to that resource's

>             provider. This change can expose bugs in the resource providers that lead to diffs being present even if the desired

Reviewer: ```suggestion
            provider. This change can expose bugs in resource providers that lead to diffs being present even if the desired
```

>             configuration matches the actual state of the resource. Set this to {{% md %}}`1`{{% /md %}} or {{% md %}}`true`{{% /md %}} to enable the old diff behavior.

Reviewer: ```suggestion
            configuration matches the actual state of the resource. Set this to <code>1</code> or <code>true</code> to enable the old diff behavior.
```

    </dt>
    <dd>
        <p>

>             Set this environment variable to opt-in to using {{% md %}}`yarn`{{% /md %}} instead of {{% md %}}`npm`{{% /md %}} for installing Node.js dependencies.

Reviewer: ```suggestion
            Set this environment variable to opt-in to using <code>yarn</code> instead of <code>npm</code> for installing Node.js dependencies.
```

        </p>
        <pre><code class="text-xs">PULUMI_PREFER_YARN=true</code></pre>
    </dd>
    <dt>
        <span class="font-mono">
            PULUMI_PYTHON_CMD
        </span>
    </dt>
    <dd>
        <p>
            As of <a href="https://github.com/pulumi/pulumi/blob/master/CHANGELOG.md#0166-2018-11-28"><code>v0.16.6</code></a>,

>             the Pulumi CLI uses {{% md %}}`python3`{{% /md %}} instead of {{% md %}}`python`{{% /md %}} when running a Python program. Set this environment variable to

Reviewer: ```suggestion
            the Pulumi CLI uses <code>python3</code> instead of <code>python</code> when running a Python program. Set this environment variable to
```

    </dd>
    <dt>
        <span class="font-mono">
            PULUMI_SKIP_UPDATE_CHECK
        </span>
    </dt>
    <dd>
        <p>
            As of <a href="https://github.com/pulumi/pulumi/blob/master/CHANGELOG.md#0179-2019-04-30"><code>v0.17.9</code></a>,
            you may skip the Pulumi version update check by setting this environment variable.
        </p>
        <pre><code class="text-xs">PULUMI_SKIP_UPDATE_CHECK=true</code></pre>
    </dd>
    <dt>
        <span class="font-mono">
            PULUMI_TEST_MODE
        </span>
    </dt>
    <dd>
        <p>
            As of <a href="https://github.com/pulumi/pulumi/blob/master/CHANGELOG.md#0177-2019-04-17"><code>v0.17.7</code></a>,

>             you can enable a new "test mode" by setting this to {{% md %}}`true`{{% /md %}} in either the Node.js or Python SDK. This mode allows you

Reviewer: ```suggestion
            you can enable a new "test mode" by setting this to<code>true</code> in either the Node.js or Python SDK. This mode allows you
```
